providers/route53.go

```diff
@@ -21,6 +21,21 @@ var (
 )
 
 func init() {
+	if len(os.Getenv("AWS_REGION")) == 0 {
```

Comment:
true. i erred on the side of modifying as little as possible to make this work, but you're right that those checks can be removed now.